web/src/components/users/RootAuthMethods.jsx

```diff
@@ -20,14 +20,26 @@
  */
 
 import React, { useState, useEffect } from "react";
-import { Skeleton, Truncate } from "@patternfly/react-core";
+import { Button, Skeleton, Truncate } from "@patternfly/react-core";
 import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
 import { Em, RowActions } from '~/components/core';
 import { RootPasswordPopup, RootSSHKeyPopup } from '~/components/users';
 
 import { useCancellablePromise } from "~/utils";
 import { useInstallerClient } from "~/context/installer";
 
+const MethodsNotDefined = ({ setPassword, setSSHKey }) => {
+  return (
+    <div className="stack">
+      <div className="bold">No root auth method defined yet</div>
+      <div>Please, define at least one root authentication method for being able to log into the system as administrative user.</div>
```

Comment:
Maybe this shorter version? "Please, define an authentication method for loggin into the system as root."